nais/topics.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kafka.nais.io/v1
+kind: Topic
+metadata:
+  name: privat-k9-dittnav-varsel-beskjed
+  namespace: dusseldorf
+  labels:
+    team: dusseldorf
+spec:
+  pool: {{kafka-pool}}
+  config: # optional; all fields are optional too; defaults shown
+    cleanupPolicy: delete  # delete, compact
+    minimumInSyncReplicas: 1
+    partitions: 1
+    replication: 3  # see min/max requirements
+    retentionBytes: -1  # -1 means unlimited
+    retentionHours: 72  # -1 means unlimited
```

Comment:
Ønsker vi kun 72 timers retention på denne topicen?

Reply:
Nei det er godt poeng. Hva burde vi ha? Det samme gjelder for så vidt for omsorgsdager-aleneomsorg også, har bare brukt default verdi der også.

Reply:
Ja, det vær være like lenge som på de vi har onprem idag. 
Kan bli en utfordring å fikse eventuelle feil under en lengre ferieperiode.

Reply:
Ja enig, fikser